test/cookbooks/python-webapp-test/recipes/default.rb

```diff
@@ -0,0 +1,23 @@
+#
+# Cookbook Name:: osl_application_python_test
+# Recipe:: default
+#
+# Copyright 2015, OSU Open Source Lab
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+file 'test' do
```

Comment:
The default recipe is essentially a placeholder. It, along with the default test suite, simply creates a file and then tests for its existence.